react/features/toolbox/components/web/Toolbox.js

```diff
@@ -376,6 +388,14 @@ class Toolbox extends Component<Props> {
                         visible = { this._shouldShowButton('camera') } />
                 </div>
                 <div className = 'button-group-right'>
+                    { this._shouldShowButton('localrecording')
+                        && <LocalRecordingButton
+                            isDialogShown =
+                                { this.props._localRecState.showDialog }
```

Comment:
Also take into account the dialog being shown but the toolbar having auto-hidden.

Reply:
I forgot that maybe the design will change.